components/mongodb/actions/search-documents/search-ducuments.mjs

```diff
@@ -0,0 +1,33 @@
+import mongodbApp from "../../mongodb.app.mjs";
+
+export default {
+  key: "mongodb-search-documents",
+  name: "Search Documents",
+  description: "Search for specific documents or return all documents. [See the docs here](https://docs.mongodb.com/manual/reference/method/db.collection.find/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mongodbApp,
+    collection: {
+      propDefinition: [
+        mongodbApp,
+        "collection",
+      ],
+    },
```

Comment:
The description for this is "Collection where the new document will be inserted". It should be something like "Collection containing the document(s) to search for."